Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp

```diff
@@ -973,84 +976,86 @@
 
       mesh_index_pair location;
 
-      if (!position_is_reachable(rx, ry)) {
+      if (!position_is_reachable(pos)) {
         SERIAL_ECHOLNPGM("(X,Y) outside printable radius.");
         return;
       }
 
       save_ubl_active_state_and_disable();
 
       LCD_MESSAGEPGM(MSG_UBL_FINE_TUNE_MESH);
-      ui.capture();                                                 // Take over control of the LCD encoder
+      ui.capture();                                         // Take over control of the LCD encoder
 
-      do_blocking_move_to(rx, ry, Z_CLEARANCE_BETWEEN_PROBES);      // Move to the given XY with probe clearance
+      do_blocking_move_to(pos, Z_CLEARANCE_BETWEEN_PROBES); // Move to the given XY with probe clearance
```

Comment:
regression BUG: Z_CLEARANCE_BETWEEN_PROBES now used as feedrate instead of a Z-coordinate.